shared/search/providers.test.ts

```diff
@@ -19,12 +19,18 @@ const spec: LanguageSpec = {
     filterDefinitions: <T extends Result>(results: T[]) => results.filter(result => result.file !== '/f.ts'),
 }
 
-const textDocument = createStubTextDocument({
+const textDocument1 = createStubTextDocument({
     uri: 'git://sourcegraph.test/repo?rev#/foo.ts',
```

Comment:
FYI (unrelated to the PR) paths after `#` do not have leading slashes

Reply:
https://github.com/sourcegraph/code-intel-extensions/pull/530